### PR TITLE
chore: update deps and fix the act() test warnings

### PR DESCRIPTION
**Pull request by rolandjitsu**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [x] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [x] Yes, I've updated the documentation
- [ ] Not relevant

**Summary**

Get rid of the dependency security warnings and of the `act(...)` warnings in tests.

**Does this PR introduce a breaking change?**

Nope.

**Other information**
Note that we need Node 10+ for any dev on the project (due to rollup), so we have to drop that from the test matrix.

### CODE REVIEW

**Comment by rolandjitsu** [comment]
@okonet I think we should either get rid of yarn or npm lock files. Doesn't make much sense to have both and yarn complains about it.

**Review by rxmarbles — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -258,7 +253,7 @@ test('invoke onDragEnter when dragenter event occurs', async () => {
       )}
     </Dropzone>
   )
-  const { container } = render(ui)
+  const { container, rerender } = render(ui)
```
Comment (donedgardo) [inline README.md:256]: in line 260 I think you have to change the arguments of flushPromises

**Review by donedgardo — CHANGES_REQUESTED** [review]
I've tried this and now I don't get the act warnings, although the test fail as dragOnEnter jest mock function is not being called.

File: README.md (hunk 1 above)
Comment (rolandjitsu) [reply README.md:256]: Thanks for reporting this. I've just updated it.

**Review by rxmarbles — APPROVED** [review]

**Pull request merged**